src/lib/components/labeledInput/labeledInput.module.scss

```diff
@@ -0,0 +1,137 @@
+@import '../../styles/variables/variables.module.scss';
+@import '../../styles/themes/themes.module.scss';
+@import '../../styles/global/reset.module.scss';
+
+.wrapper {
+  display: inline-block;
+  font-size: 16px;
+  position: relative;
+  width: 100%;
+  margin-top: 40px;
+
+  // &.noLabel {
+  //   margin-top: 10px;
+  // }
+
+  // &.topLabel {
+  //   margin-top: 20px;
+  // }
+
+  textarea {
+    min-height: 100px;
+  }
+
+  textarea,
+  input {
+    -webkit-appearance: none;
+    border: 0;
+    outline: none;
+    width: 100%;
+
+    @include themify() {
+      color: themed('fieldColor');
+      background-color: themed('fieldBackground');
+      border-bottom: 1px solid themed('borderColor');
+      font-family: themed('primaryFontFamilySemiBold');
+      font-size: themed('fieldFontSize');
+      font-weight: themed('fieldFontWeight');
+      padding: themed('inputPadding');
+    }
+
+    &[placeholder] ~ label {
+      transform: translate3d(0, -24px, 0) scale(0.75);
+    }
+
+    &[required] ~ label::after {
```

Comment:
Remember that we never put `required` in the HTML as we use our custom Validations

Reply:
Added required class too